docs/DeveloperGuide.adoc

```diff
@@ -780,15 +780,28 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 [appendix]
 == Product Scope
 
-*Target user profile*:
+*Target user profile:*
 
-* has a need to manage a significant number of contacts
+* is currently a student of NUS
+* is in a relationship with a student of NUS
+* needs to remember special moments
+* needs to remember special dates
+* needs to schedule meetings
+* needs to find a good time to meet
```

Comment:
This point is very related to the previous point, isn't it? If so, you can combine them..